docs/wiki/ModulesUsage.md

```diff
@@ -307,3 +314,105 @@ The example assumes you are using a [`bicepconfig.json`](https://docs.microsoft.
     }
 }
 ```
+
+## Solution deployment
+
+The bicep modules provided by this repo can also be clubbed together to create more complex infrastructures and in the form a completely reusable solutions or products.
```

Comment:
```suggestion
The modules provided by this repo can be orchestrated to create more complex infrastructures and as such reusable solutions or products.
```

Reply:
Overall, I', not sure if that fit's here. The section should be `Pipeline-orcehstration` and be a counter-part to the explaination what template-orchestration is (above).

Then, there should be a sub-header `# Multi-Repository approach` which explicitely explains this flavor